CHANGES.rst

```diff
@@ -67,6 +67,11 @@ astropy.table
 astropy.tests
 ^^^^^^^^^^^^^
 
+- The function ``quantity_allclose`` was moved to the ``units`` package with the new,
+  shorter name ``allclose``. It is deprecated in ``tests``. [#7252]
```

Comment:
I think the last sentence should be something like 'The old import will continue to work but may be deprecated in future'